src/Microsoft.IdentityModel.Tokens/LogMessages.cs

```diff
@@ -28,12 +28,26 @@
 // Microsoft.IdentityModel.Tokens
 // Range: 10000 - 10999
 
+using System.Runtime.CompilerServices;
```

Comment:
@mafurman while there is no harm putting this here, we usually put it in AssemblyInfo.cs.
Can you move it there?

see: https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/blob/dev/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Properties/AssemblyInfo.cs#L53

Reply:
Just made the change!